### PR TITLE
modified meta tags and column selection syntax for keyworld collision

### PR DESCRIPTION
**Pull request by jchenff**

### CODE REVIEW

File: models/src_models_zendesk.yml [hunk 1]
```diff
@@ -131,15 +134,16 @@ models:
         description: Boolean field indicating if the comment is a twitter tweet
       - name: is_voice_comment
         description: Boolean field indicating if the comment is a voice comment
-      - name: from
+      - name: from_number
         meta:
           sensitive: true
-      - name: to
+      - name: to_number
         meta:
```
Comment (aaron-ortega) [inline models/src_models_zendesk.yml]: `to_number` is a snowflake [function](https://docs.snowflake.com/en/sql-reference/functions/to_decimal.html) we might want to change it to something else

Comment (jchenff) [reply models/src_models_zendesk.yml]: ok, makes sense

**Review by mduchak — APPROVED** [review]
Looks good.

File: models/src_models_zendesk.yml (hunk 1 above)
Comment (jchenff) [reply models/src_models_zendesk.yml]: fixed

**Pull request merged**